actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -687,6 +683,18 @@ def time_tag(date_or_time, *args, &block)
 
         content_tag("time".freeze, content, options.reverse_merge(datetime: datetime), &block)
       end
+
+      private
+
+        def normalize_distance_of_time_argument_to_time(value)
+          if value.is_a?(Numeric)
+            Time.at(value)
+          elsif value.respond_to?(:to_time)
+            value.to_time
+          else
+            raise ArgumentError, "#{v} must be convertible to_time"
```

Comment:
This needs to be `raise ArgumentError, "#{value} must be convertible to_time"` - getting a `NameError` when running the tests.